Update `README.md` files. (#602)
linebender/vello: vello/README.md

It is used as the rendering backend for [Xilem], a Rust GUI toolkit.

> ⚠️ WARNING
>

> > Vello can currently be considered in an alpha state. In particular, we're still working on the following:

Reviewer: Is this release good enough to call beta? I think probably not, but it's plausible, so I thought I'd check

Author: Probably not, pending some more robustness work.

surface_texture.present();
```

> See the [`../examples`](https://github.com/linebender/vello/tree/main/examples) directory to see how that code integrates with frameworks like winit and bevy.

Reviewer: I think this probably likely to be more/just as confusing - I'd suggest maybe:
```suggestion
The [`examples`](https://github.com/linebender/vello/tree/main/examples) folder in the repository shows that code integrated with frameworks like winit and bevy.
```

The specific issue is that this README is intended to be read on crates.io and docs.rs, with the GitHub use cases only peripherally

Author: I simplified it and removed the confusing dots.

cargo apk run -p with_winit
```

> 💡 TIP

> >

Reviewer: It might be worth removing the web/android/with_winit detailed running instructions from this crate-level readme

I think that should be a post-release follow-up, probably.

## Community

Discussion of Vello development happens in the [Linebender Zulip](https://xi.zulipchat.com/), specifically the [#gpu stream](https://xi.zulipchat.com/#narrow/stream/197075-gpu).
All public content can be read without logging in.

Contributions are welcome by pull request.
The [Rust code of conduct] applies.

> Unless you explicitly state otherwise, any contribution intentionally submitted for inclusion in the work by you, as defined in the Apache 2.0 license, shall be licensed as noted in the [License](#license) section, without any additional terms or conditions.

Reviewer: For android_trace, I excluded this sentence from the crate level README, as it is only really meaningful at a repository level.

Author: Crates can have different licenses, so there is some potential value there. I don't care about this specific sentence much, because it doesn't change anything legally, and the same sentence is already present in the Apache license.

I guess I'm fine with only having it in the repo level readme.
